api/v1beta1/horizontest_types.go

```diff
@@ -173,6 +175,10 @@ func init() {
 	SchemeBuilder.Register(&HorizonTest{}, &HorizonTestList{})
 }
 
+func (instance HorizonTest) IsReady() bool {
+       return instance.Status.Conditions.IsTrue(condition.DeploymentReadyCondition)
```

Comment:
When we were discussing this task in the office I remember you saying InputReadyCondition is not relevant to us. But I could be wrong and in that case we can use the AllsubConditionsTrue function, because I was just leaving out the InputReadyCondition and putting other conditions into IsReady function.

Reply:
Yes, true. What we want to do is to remove the `InputReadyCondition` from the conditionList as well. And from any code that references it [1].

[1] https://github.com/openstack-k8s-operators/test-operator/pull/135/files#diff-763e3c6d751fb09d0fb537d31b313f6bb32dea43ee46573de067f97e09a477e2R122

Reply:
Alright, I have deleted all of InputReadyCondition mentions.